app/locales/en.json

```diff
@@ -1,12 +1,139 @@
 {
-	"Postal code": "Postal code",
-	"Important": "Important",
-	"Page not found": "Page not found",
-	"We can't find the page or service you're looking for": "We can't find the page or service you're looking for",
-	"The link you clicked may be broken, or the page no longer exists": "The link you clicked may be broken, or the page no longer exists",
-	"Something went wrong!": "Something went wrong!",
-	"Something must have gone wrong": "Something must have gone wrong",
-	"Send an e-mail to": "Send an e-mail to",
-	" if you can't get it to work.": " if you can't get it to work.",
-	"Share on": "Share on"
-}
\ No newline at end of file
+  "Postal code": "Postal code",
```

Comment:
Maybe checkout this whole file?

Reply:
I think I fixed that—I had to refresh the PR for it to show up, though.

Reply:
Cool!